examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -158,14 +156,151 @@ function debugResetExternalData(): void {
 // themselves (as if they were editing it outside of Fluid).
 // TODO: Consider how we might simulate errors/failures here to play with retry and recovery.
 const ControlsView: React.FC<IControlsViewProps> = (props: IControlsViewProps) => {
-    console.log(props.model);
     return (
         <div>
             <h3>Debug controls</h3>
             <div style={{ margin: "10px 0" }}>
                 <button onClick={ debugResetExternalData }>Reset external data</button><br />
-                <button onClick={ props.model.debugSendCustomSignal }>Trigger external data change signal</button><br />
             </div>
         </div>
     );
 };
+
+
+interface ITaskRowProps {
+    task: ExternalDataTask
+}
+
+/**
+ * The view for a single task in the TaskListView, as a table row.
+ */
+const TaskRow: React.FC<ITaskRowProps> = (props: ITaskRowProps) => {
+    const { task } = props;
+
+    const IdChangeHandler = (e: React.SyntheticEvent<HTMLInputElement>): void => {
```

Comment:
Nit: I think the handlers want to be named in `camelCase`, since they're non-React-component functions.